### PR TITLE
Translate README

### LINKED ISSUE #5: Translate README to English
As this repository is public and available to all, we need to update the README.

### PR DESCRIPTION
**Pull request by timharek**
A simple translation of our readme with new instructions and details about deployment etc.

Fixes #5.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,86 +1,82 @@
...
+1. Clone the repository, `git clone git@github.com:fribyte-code/fribyte.no.git`.
+2. Run `zola serve` to run Zola locally.
 
-Når man dytter til `main` så deployes dette live på
-[fribyte.no](https://fribyte.no). Nettsiden bor på VM'en **nettsider** under
-Konrad.
+## Deployment
 
-## Bruk
+- The site is self-hosted on friBytes serveres. It's running Pengebingen.
```
Comment (lucafossen) [inline README.md:12]: Suggestion:
The site is self-hosted on friByte's servers (It's running on Pengebingen).

File: README.md [hunk 2]
```diff
@@ -1,86 +1,82 @@
...
+### News
 
-For å publisere nyheter så lager man bare ett nytt dokument under
-`content/nyheter`. Et eksempel kan være `content/nyheter/2022-01-01-godt-nyttar.md`.
-Hvis denne nyheten skal inneholde bilder osv. er det nødvendig å lage en
-mappe istedenfor, for å forsette på forrige eksempel
-`content/nyheter/2022-01-01-godt-nyttar/index.md`. Nå er det `index.md` som
-holder selve "artikkelen", plasser alt av bilder og slik i samme mappe.
+In order to publish news we need to create a new markdown-file under
+`content/nyheter`. An example can be
```
Comment (lucafossen) [inline README.md:58]: Suggestion: "For example:"

**Review by lucafossen — COMMENTED** [review]
Nice! I have suggested a few nitpicks for readability.

File: README.md (hunk 1 above)
Comment (mathiash98) [reply README.md:12]:
```suggestion
- The site is self-hosted on friByte'ss serveres (It's running Pengebingen).
```

Test out the "Add a suggestion" feature @lucafossen 

image

Then you will get these nice suggestions that can be automatically committed:

image


And soon be pro in Code review 😎

Comment (lucafossen) [reply README.md:12]: Ah, nice! I did this on the iOS application, not sure if the button exists there..?
will get on desktop asap and try it out

**Review by mathiash98 — APPROVED** [review]

**Pull request merged**